packages/utilities/psammead-locales/moment/index.stories.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Can this disable be more specific? Maybe disable the es5 only stuff?

Reply:
had to disable eslint here for the entire file .i think its better if we maintained the ES6 features in here, otherwise it will force us into writing this file into ES5